arkouda_proxy_client/arkouda_proxy_client/arkouda_pb2_grpc.py

```diff
@@ -0,0 +1,66 @@
+# Generated by the gRPC Python protocol compiler plugin. DO NOT EDIT!
+"""Client and server classes corresponding to protobuf-defined services."""
+import grpc
+
+from . import arkouda_pb2 as arkouda__pb2
+
+
+class ArkoudaStub(object):
+    """Missing associated documentation comment in .proto file."""
```

Comment:
Not a deal breaker, but should we go ahead and add the docstring to `.proto`? I am just thinking it may be useful for anyone who is not familiar with working with gRPC or any messaging framework.

There are a few other places this appears. Again, not a big deal. I just think it could prove useful long term.

Reply:
@Ethan-DeBandi99 arkouda_proxy_server has a protos directory so that the server stubs/classes can be generated and arkouda_proxy_server has a proto directory for the corresponding client stub/classes. I could potentially put the arkouda.proto file in one central place.

Reply:
@hokiegeek2 - I think this addressed my question above. I resolved that. Here I was asking if we should add the docstrings it says are missing?

Reply:
@Ethan-DeBandi99 ah, I see, okay, I'll check it out.